plugins/woocommerce/includes/abstracts/abstract-wc-order.php

```diff
@@ -2175,6 +2175,47 @@ public function get_formatted_line_subtotal( $item, $tax_display = '' ) {
 		return apply_filters( 'woocommerce_order_formatted_line_subtotal', $subtotal, $item, $this );
 	}
 
+	/**
+	 * Gets item subtotal - formatted for display.
+	 *
+	 * This method returns the formatted unit price for an order item.
+	 *
+	 * @param object $item Item to get unit price from.
+	 * @param string $tax_display Incl or excl tax display mode.
+	 * @return string Formatted item subtotal.
+	 */
+	public function get_formatted_item_subtotal( $item, $tax_display = '' ) {
```

Comment:
Similar concern here on the file size and this functionality is relevant to emails only.

So, WDYTH about relocating this under a new class in `plugins/woocommerce/src/Internal/Email` (naming wise can be `EmailFragmentsGenerator` or something similar)?

Reply:
Good point, I was hesitant adding to this already huge class before as well. I moved these two functions & corresponding test cases to new helper classes in https://github.com/woocommerce/woocommerce/pull/57573/commits/481c340fedb15dfd154b64a75761f81b7bbfa585:

- `get_formatted_item_subtotal` -> `/src/Internal/Email/OrderPriceFormatter`
- `is_pos_order` -> `/src/Internal/Orders/POSOrderUtil`

Please feel free to suggest better naming/location.